src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -680,7 +680,8 @@
       (= subject-alt-names-oid oid)
       (and (utils/subtree-of? ppAuthCertExt oid) (not (= cli-auth-oid oid)))
```

Comment:
Looks like this is already here?

Reply:
oh hey look. Apparently reading is hard. So it just needs to be added to the ruby?

Reply:
Is the ruby being actively maintained or is that stuff deprecated? I ask because it seems to have not been updated since 2016.. but I'm still new to this repo soooo ¯\_(ツ)_/¯

Reply:
Yes we still need it. It hasn't changed because it hasn't needed to.